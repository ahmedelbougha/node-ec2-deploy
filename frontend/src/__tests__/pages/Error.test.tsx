import renderer from 'react-test-renderer';
import { createBrowserRouter, RouterProvider } from 'react-router-dom';
import ErrorPage from '../../pages/Error';

const router = createBrowserRouter([
  {
    // dashboard page
    path: '/',
    element: <ErrorPage />,
  },
]);

describe('Error Page', () => {
  it('renders', () => {
    const component = renderer.create(<RouterProvider router={router} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
