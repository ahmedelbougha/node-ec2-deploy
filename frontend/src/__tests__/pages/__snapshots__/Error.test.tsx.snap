// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page renders 1`] = `
<div
  className="sc-bcXHqe CLyot"
>
  <div>
    <h1>
      Oops!
    </h1>
    <p>
      Sorry, an unexpected error has occurred.
    </p>
    <p>
      <i>
        
      </i>
    </p>
  </div>
</div>
`;
